Spielregeln.md

```diff
@@ -0,0 +1,88 @@
+# Spielregeln
+
+## Vorwort
+
+Bierball ist ein Sport, in welchem der Spaß aller Spielenden im Vordergrund steht. 
+Alle Teilnehmenden sind zu jeder Zeit mit Respekt zu behandeln.
+Von allen Teilnehmenden wird Fair-Play erwartet.
+
+## § 1 Spielziel
+
+(1) Das Spiel gilt als beendet, sobald ein Team die Getränke komplett geleert hat.
+
+(2) Ein Getränk gilt als geleert, wenn sich das Trinkbehältnis vom Spielenden mit Öffnung nach unten über den Kopf gehalten wurde, ohne dass eine nennenswerte Menge an Flüssigkeit aus dem Behältnis entweicht.
+Sollte zu viel Flüssigkeit entweichen, kann dies mit einem Strafschluck (siehe § 6 (1)) sanktioniert werden.
+
+## § 2 Teams
+
+(1) Gespielt wird mit zwei gleich großen Teams, die jeweils aus mindestens einer Person bestehen.
+
+## § 3 Spielgeräte
+
+(1) Die folgenden Spielgeräte werden zum Spielen benötigt:
+
+1. Ein Wurfgerät
+2. Ein Ziel
+3. Ein Getränkebehältnis pro spielender Person
+
+(2) Das Wurfgerät ist ein annähernd runder Gegenstand, der sich leicht einhändig werfen lässt.
+
+(3) Das Ziel muss in Größe und Form einer 1,0-Liter-Plastikflasche ähneln.
+Es muss in einer Art beschwert sein (z. B. durch das Befüllen mit Wasser), dass es nicht durch leichte Krafteinwirkung, wie z. B. Windstöße umfällt.
+Allerdings ist sicherzustellen, dass es möglich ist, das Ziel mit dem Wurfgerät zu Fall zu bringen.
+
+(4) Getrunken wird aus Glasflaschen der 0,5-Liter-Klasse, deren Öffnung DIN EN ISO 12822 oder DIN EN ISO 12821 (handelsübliche Bierflasche mit Kronkorkenverschluss) entspricht.
+Die Getränke müssen Kohlensäure enthalten.
+Es ist explizit erlaubt, Getränke zu konsumieren die keinen Alkohol enthalten bzw. nicht als Bier klassifiziert werden.
+
+## § 4 Spielfeld
+
+(1) In der Mitte des Spielfeldes wird das Ziel aufgestellt.
+
+(2) Von dort ausgehend sind in entgegengesetzte Richtung 6 m (~7 Schritte) abzumessen an denen sich die jeweiligen Grundlinien (auch "Wurflinien") der zwei Teams befinden.
+
+(3) Die Grundlinien der Teams verlaufen parallel.
+
+## § 5 Ablauf
+
+(1) Es gibt immer genau ein Team, welches versucht das Ziel mithilfe des Wurfgeräts zu Fall zu bringen ("werfendes Team") und ein Team, welches darauf reagiert ("aufstellendes Team").
+Nach jedem Wurf wird gewechselt.
+
+(2) Vor Beginn entscheidet ein Münzwurf oder eine Partie Schere-Stein-Papier zwischen zwei Mitgliedern der spielenden Teams über den ersten Wurf.
+Das gewinnende Team entscheidet, welches Team zuerst wirft.
+
+(3) Vor jedem Wurf, muss sichergestellt werden, dass das gegnerische Team bereit ist.
```

Comment:
`Dies geschieht mit dem Heben einer Hand oder mit einem Zuruf durch das werfende Team, was vom aufstellenden Team durch ein Handheben oder einen zweiten Ruf bestätigt wird.`

So könnte ein für alle Mal sichergestellt werden, dass beide Teams bereit sind. Allerdings kann das schnell in Streit über derartige Kleinigkeiten enden, wenn neue Mitspieler die Regel nicht gewöhnt sind.